app/dependencies.php

```diff
@@ -91,9 +92,26 @@ function (ContainerInterface $c): Auth\DonationPublicAuthMiddleware {
             return new Chatter($transport);
         },
 
+        StripeChatterInterface::class => static function (ContainerInterface $c): ChatterInterface {
+            /**
+             * @var array{
+             *    notifier: array{
+             *      slack: array{
+             *        stripe_channel: string
+             *      }
+             *    }
+             *  } $settings
+             */
+            $settings = $c->get('settings');
+            $stripeChannel = $settings['notifier']['slack']['stripe_channel'];
+            /** @var SlackChannelChatterFactory $chatterFactory */
+            $chatterFactory = $c->get(SlackChannelChatterFactory::class);
+            return $chatterFactory->makeChatter($stripeChannel);
+        },
+
         SlackChannelChatterFactory::class => static function (ContainerInterface $c): SlackChannelChatterFactory {
+            /** @var array $settings */
             $settings = $c->get('settings');
```

Comment:
Why the change from `assert` to `@var`? Seem to be going in the wrong direction, away from type safety.

Reply:
I guess no great reason except that it was closer to the other psalm-placating changes I was thinking about, and as discussed `$settings` being a generic array will likely be temporary anyway.